debian/patches/0002-Add-buildall.sh.patch

```diff
@@ -0,0 +1,155 @@
+From: john spurling <john.spurling@reddit.com>
```

Comment:
So I'm concerned with this because it introduces code duplication from other projects and it loses a lot of the advantages of building with debian's system.

What about:
* Make a new PPA for mcrouter (due to launchpad's restriction on only keeping the latest version of a package)
* Build all of mcrouter's specific dependencies there.
* Modify configure.ac—as you did—to link statically

@spladug thoughts?